.github/workflows/mps-model-adapters.yaml

```
@@ -0,0 +1,34 @@
+name: MPS compatibility
+
+on:
+  push:
+    branches:
+      - '**'
```

```suggestion
      - 'main'
```
`**` will cause the workflow to trigger twice on pushes in PRs

---

There is an action `skip-duplicate-actions` that avoids that and still allows to build branches without having to create a PR.